doc/cprover-manual/api.md

```diff
@@ -211,6 +211,57 @@ returns true when it is safe to do both.  These predicates can be given an
 optional size; when the size argument is not given, the size of the subtype
 (which must not be **void**) of the pointer type is used.
 
+#### \_\_CPROVER\_havoc\_object
```

Comment:
Random place for comment: it would be great if the commit message also answered the "Why?" question. See https://chris.beams.io/posts/git-commit/ as linked from the pull-request template. On this occasion, it would be good to answer the "why did `__CPROVER_havoc_object` not suffice" question.